app/models/badge.rb

```diff
@@ -305,6 +305,10 @@ def image_url
     end
   end
 
+  def for_beginners?
+    id == Welcome || (badge_grouping_id == BadgeGrouping::GettingStarted && id != NewUserOfTheMonth)
```

Comment:
This might bites us in the future if/when we add a new badge in the `GettingStarted` group.

I think being explicit would be the best way. Something like

```ruby
BEGINNER_BADGE_IDS ||= [
  Welcome,
  Reader,
  Autobiographer,
  Editor,
  WikiEditor,
  FirstLike,
  # ...
]

def for_beginners?
  BEGINNER_BADGE_IDS.include?(id)
end

Reply:
Yeah, it's a good point, but there exists another consideration. Some badges are from plugins. For example <kbd>First Reaction</kbd>, <kbd>Licensed</kbd> and <kbd>Certifed</kbd>. I think we cannot add these to the list in Core.

If we suppress all the Getting Started badges these badges from plugins would be suppressed automatically. 

> This might bites us in the future if/when we add a new badge in the GettingStarted group.

This is how it was intended to work at the very beginning - we wanted to suppress all the Getting Started badges.

What if leave it like this by now. In the future, we always can switch to an explicit list or even to a new field in the database table how it was implemented in previous closed PR.

Reply:
This PR only adds one new badge to list - <kbd>Welcome</kdb>. The rest of the logic still working as before.

Reply:
We can add an API for plugins to add new beginner badges, but sure, we can start with this.